src/test/java/uk/gov/hmcts/cmc/claimstore/processors/NoCacheFilterTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.cmc.claimstore.processors;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.io.IOException;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+@ExtendWith(MockitoExtension.class)
+class NoCacheFilterTest {
+    @Mock
+    private FilterChain chain;
+
+    @Mock
+    private HttpServletRequest request;
+
+    @Mock
+    private HttpServletResponse response;
+
+    private NoCacheFilter filter;
+
+    @BeforeEach
+    void setUp() {
+        filter = new NoCacheFilter();
+    }
+
+    @Test
+    void shouldAddHeaders() throws ServletException, IOException {
+        filter.doFilterInternal(request, response, chain);
+        assertAll(
+            () -> verify(response).addHeader("pragma", "no-cache"),
+            () -> verify(response).addHeader("Cache-Control", "no-store"),
+            () -> verifyZeroInteractions(request)
```

Comment:
looks like `verifyZeroInteractions` is deprecated. `verifyNoMoreInteractions` can be used which has same behaviour.

Reply:
Thanks, that didn't show up in my IDE for some reason. I'm changing it to `verifyNoInteractions()` which I feel communicates the sentiment better